battleship.py

```diff
@@ -80,7 +99,14 @@ def createShip():
 Returns: bool
 '''
 def checkShip(grid, ship):
-    return
+    count=0
+    for i in ship:
+        if grid[i[0]][i[1]]==1:
+            count+=1
+            if count==3:
+              return True
+        else:
+            return False

```

Comment:
Reduce the number of lines in the function by checking the not equal condition and returning False, and returning True on completion of the loop.